README.md

```diff
@@ -18,8 +18,8 @@ If there is no data available, crawl from the website, then analyse data from lo
 > **First Rule: BE GOOD AND TRY TO FOLLOW A WEBSITE’S CRAWLING POLICIES. Don't crawl the website to often!**
 
 ## wiki
-- https://puppylpg.github.io/python/csgo/2019/12/01/python-crawler-buff.html;
-- https://puppylpg.github.io/python/csgo/2019/12/07/python-crawler-buff-optimaze.html;
+- https://puppylpg.github.io/2019/12/01/python-crawler-buff/
```

Comment:
https://puppylpg.github.io/2019/12/02/python-crawler-buff/

Reply:
I will fix this in the following commit.